CMake/resolve_dependency_modules/README.md

```diff
@@ -1,7 +1,47 @@
+# Dependency List
+Following is the list of libraries and their minimum version
+that Velox requires. Some of these libraries can be installed
+via a platform's package manager (eg. `brew` on MacOS).
+The versions of certain libraries is the default provided by 
+the platform's package manager. Some libraries can be bundled
+by Velox. See details on bundling below.
+
+| Library Name      | Minimum Version | Bundled? | 
+|-------------------|-----------------|----------|
+| ninja             | default         | No       |
+| ccache            | default         | No       |
+| icu4c             | default         | Yes      |
+| gflags            | default         | yes      |
+| glog              | default         | Yes      |
+| gtest (testing)   | default         | Yes      |
+| libevent          | default         | No       |
+| lz4               | default         | No       | 
+| snappy            | default         | No       |
+| lzo               | default         | No       |
+| xz                | default         | No       |
+| zstd              | default         | No       |
+| openssl           | default         | No       |
+| protobuf          | 21 (exact)      | Yes      |
+| boost             | 1.66.0          | Yes      |
+| flex              | 2.5.13          | No       |
+| bison             | 3.0.4           | No       |
+| cmake             | 3.14            | No       |
+| double-conversion | 3.1.5           | No       |
+| xsimd             | 10.0.0          | Yes      |
+| re2               | 2021-04-01      | No       |
+| fmt               | 10.1.1          | Yes      |
+| folly             | v2023.12.04.00  | Yes      |
+| fizz              | v2023.12.04.00  | No       |
+| wangle            | v2023.12.04.00  | No       |
+| mvfst             | v2023.12.04.00  | No       |
+| fbthrift          | v2023.12.04.00  | No       |
```

Comment:
Why are these not bundled?

Reply:
These got added for the Remote Function Feature which is not enabled by default. We will add bundling for them.